addons/overheating/XEH_preInit.sqf

```diff
@@ -4,4 +4,14 @@ ADDON = false;
 
 #include "XEH_PREP.hpp"
 
+FakeCBA_fnc_getMagazineIds = {
```

Comment:
Not a fan of this - rather use an `if isnil` check and use the actual name instead.